### ISSUE TITLE
Feature request: Generate code coverage summary information similar to solidity-coverage

### ISSUE THREAD

**Issue by banescusebi** [dapphub/dapptools#504]
At the moment `hevm dapp-test --coverage` outputs the source code files with some symbols in front of each line, which indicates if those lines were covered or not, or if they can't be covered. However, it does not give an indication of what percentage of instructions were covered or what the branch coverage is.

The percentages mentioned above, which some people find quite useful, are given by solidity-coverage in the form of a table (see attached image) 
![image](https://user-images.githubusercontent.com/390345/92638921-5ec5c380-f2db-11ea-8924-dae749169f84.png)

If there are more people out there who would like to get that information from this tool, let's make it known to the team. Hopefully they will implement it when/if they find the time for it.

**Comment by lucas-manuel** [comment]
Yes please! I would love to see this. Would also increase adoption of the framework a lot.

**Comment by TransmissionsDev** [comment]
i mentioned this is dappchat but putting here as well

instead of focusing on just impl'ing the specific chart, probably best to export coverage in a common format (lcov, etc) and then there are all sorts of other tools and sites to display these kind of metrics
